bcda/models/postgres/repository.go

```diff
@@ -117,9 +117,9 @@ func (r *Repository) GetCCLFFileExistsByName(ctx context.Context, name string) (
 	return true, nil
 }
 
-func (r *Repository) GetLatestCCLFFile(ctx context.Context, cmsID string, cclfNum int, importStatus string, lowerBound, upperBound time.Time, fileType models.CCLFFileType) (*models.CCLFFile, error) {
+func (r *Repository) GetLatestCCLFFile(ctx context.Context, cmsID string, cclfNum int, importStatus string, lowerBound time.Time, upperBound time.Time, fileType models.CCLFFileType) (*models.CCLFFile, error) {
```

Comment:
👋 Not sure if this change is just for readability and my comment is not a request to change it: You can specify a type for a function parameter once, if there are multiple parameters of the same type. 

Ex: In this function:`func FooString(int x, foo, bar, hello, string)` foo, bar, hello would be of the type `string`.